models/transactional.py

```diff
@@ -39,7 +39,7 @@ class ParticipantProgram(Base):
     program_cip = Column(Integer, ForeignKey('program.program_cip'), primary_key=True)
     entry_date = Column(Date, nullable=False)
     exit_date = Column(Date)
-    enrolled = Column(Boolean, nullable=False, default=False)
+    enrolled = Column(Boolean, nullable=False, default=True)
```

Comment:
From what I could see, participants are initially captured because they enroll & the status may change because of drop-outs from programs etc.